client/src/main/proto/spine/base/command.proto

```diff
@@ -95,7 +95,7 @@ message CommandContext {
 // Enumeration of possible failures when validating a command.
 enum CommandValidationError {
     // Reserved value.
-    UNKNOWN = 0;
+    UNDEFINED_COMMAND_VALIDATION_ERROR = 0;
```

Comment:
The enum constant with zero value is the default value in Protobuf messages, so it is rather  convenient to define it this way (I've faced this in Lobby). I didn't know that zero is reserved value, I'll read docs.